extensions/wikia/ExactTargetUpdates/client/ExactTargetResourceEnum.php

```diff
@@ -7,6 +7,8 @@ class ResourceEnum {
 	const CUSTOMER_KEY_USER_GROUPS = 'user_groups';
 	const CUSTOMER_KEY_USER_ID_WIKI_ID = 'UserID_WikiID';
 	const CUSTOMER_KEY_USER_PROPERTIES = 'user_properties';
+	const CUSTOMER_KEY_WIKI_LIST = 'city-list'; // TODO: Resolve prod key 'city_list'
+	const CUSTOMER_KEY_WIKI_CAT_MAPPING = 'city-cat-mapping'; // TODO: Resolve prod key 'city_cat_mapping'
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=php%3AS1135)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=php%3AS139)